internal/storage/v2/grpc/proto/storage.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package jaeger.storage.v2;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+import "opentelemetry/proto/trace/v1/trace.proto";
+
+option go_package = "storage_v2";
```

Comment:
@yurishkuro do we want the package name to be this? or should it just be `storage`?

Reply:
`package jaeger.storage.v2;` is the important thing. Go package doesn't matter much, but you would want to be something that will result in a sensible location of the generated file.